Sources/Views/EpisodeVideoView.swift

```diff
@@ -44,6 +44,21 @@ window.addEventListener("load", function (event) {
     player.setCurrentTime(time);
     if (play) { player.play(); }
   }
+
+  var lastSeenPercent = 0
+  player.on('timeupdate', function(data) {
```

Comment:
this notifies us every ~250ms of the current video play times. I use this to determine when the video progress percentage increases by 1 or more, and if it does I fire off an API request